src/pages/guidelines/color/overview.mdx

```diff
@@ -43,7 +45,7 @@ library in the context of dark and light themes.
 
 <GifPlayer color="dark">
 
-![UI color alternatives](images/introduction.gif)
+![6 screenshots showing a range of color schemes and considerations for text and data visualizations](images/introduction.gif)
 ![UI color alternatives](images/introduction.jpg)
```

Comment:
This image seems to be missing? When I pause the gif there's a missing image on the page.

Reply:
oh weird, source was `.jpeg` not `jpg`